book_source/source/module-9-miscellaneous-topics/lesson-27-reading-ethics-in-data-science/case-study-1-tracking-for-safety.md

```diff
@@ -0,0 +1,45 @@
+
+Now, Tectronix claims (see tweet above) that there is no privacy concerns here since they collected the data with users' consent and kep the data anonymous. While we assume this is true, there are still a couple potential gray-areas here.
+
+1. **Anonymous data sometimes isn't**: In the [next module](/module-10-beyond-163/lesson-29-privacy/index), we'll discuss the challenges of keeping anonymous data truly anonymous. You'll find that it's surprisingly easy, given enough information, to narrow down a large dataset with anonymous data, down to a single individual that you are looking for. In that lesson, we will talk about some techniques to ensure anonimity, but the key point is that it's challenging and rarely sufficient to just remove the users' name from the data. You can also see an example linked in [](content:module9:lesson27:case-study-1:further-resources) below on how the New York Times could identify indiviudals from a large, anonymous dataset.
+2. **What did the users consent to?** While the users might have consented to the collection of their data, the details of that consent are fairly important for us to considr whether using their data in this way is appropriate.
+
+   <br />
+
+   Many of us have probably ehad the experience of downloading the new app, and clicking through the "Terms of Conditions" as fast as we can just so we can get to the good stuff. This usually means by clicking the "I agree" button, you are (usally) consenting to the company using your data in a very generous manner without many restrictions. While we aren't lawyers, we assume many uses of data, including publishing anonymous maps like this, do not pose any _legal_ complications. So if it's legal, does that make it totally fine? Many privacy experts agree that legality is not enough, since our current laws might not accurately reflect the impact and importance of data. In fact, the European Union recently overhauled their data privacy laws with the introduction of [GDPR](https://gdpr-info.eu/); a move that vastly increased user's privacy rights that companies with European customers must abide by.
+
+   <br />
+
+   There is an additional question here of exactly what the scope of consent user's agree to. An example (mentionedin the New York Times article linked below): If a user downloads a weather app, and consents that the company use their location data to improve their ability to provide a better service for the at customer, is it allowed for that weather app to sell location data to advertisers? Some might claim that the advertising brings in money, that lets the developers of the app improve the software they use. But others might claim that the user was really giving a much narrower consent that might make it an _ethical_ issue for companies to use that data in scopes where consent was not provided.
```

Comment:
"mentionedin" -> "mentioned in"
"better service for the at customer" - extra "at"